src/components/cart/cart.tsx

```diff
@@ -32,33 +31,18 @@ export const Cart = () => {
         } else {
             alert('checkout is not configured');
         }
-    }
+    };
 
     return (
         <Drawer title="Cart" onClose={() => setIsOpen(false)} isOpen={isOpen}>
-            {isEmpty ? (
-                <div className={styles.emptyCart}>Cart is empty</div>
-            ) : (
-                <div className={styles.cart}>
-                    <div className={styles.items}>
-                        {cart?.lineItems?.map((item) => (
-                            <CartItem key={item._id} cartItem={item} />
-                        ))}
-                    </div>
-                    <div className={styles.subtotalCheckout}>
-                        {checkoutAttempted && someItemsOutOfStock && (
-                            <div className={styles.errorMessage}>Some items are out of stock</div>
-                        )}
-                        <label className={styles.subtotalLabel}>
-                            <span>Subtotal:</span>
-                            {cartTotals?.priceSummary?.subtotal?.formattedConvertedAmount}
-                        </label>
-                        <button className={classnames('primaryButton', styles.checkoutButton)} onClick={checkout}>
-                            Checkout
-                        </button>
-                    </div>
-                </div>
-            )}
+            <CartView
+                cart={cart!}
+                cartTotals={cartTotals}
+                errorMessage={error}
+                onCheckout={handleCheckout}
+                onItemRemove={(id) => removeItem(id)}
+                onItemQuantityChange={(id, quantity) => updateQuantity({ id, quantity })}
```

Comment:
let's change type of prop onItemQuantityChange to accept updateQuantity without any param changes so we can write

`onItemQuantityChange = {updateQuantity }`